src/qibolab/instruments/simulator/emulator_test/default_q0/ibmfakebelem_q0_runcard.yml

```diff
@@ -0,0 +1,53 @@
+nqubits: 5
```

Comment:
This file is just duplicating parameters.json and should be removed.

Reply:
The yaml files are there for convenience of adjusting the runcard parameters and are not actually directly used by the emulator. I will remove from the branch and just have the yaml in my local repo.

Reply:
Fixed in e035cb3 and 297f8f8